app/pages/lab-pages-editor/components/TasksPage/components/StepItem/BranchingControls.jsx

```diff
@@ -0,0 +1,77 @@
+export default function BranchingControls({
+  allSteps = [],
+  task,
+  taskKey,
+  updateAnswerNext = () => {}
+}) {
+  if (!task || !taskKey) return null;
+
+  const answers = task.answers || []
+
+  function onChange(e) {
+    const next = e.target?.value;
+    const index = e?.target?.dataset.index;
+    updateAnswerNext(taskKey, index, next);
+  }
+
+  return (
+    <ul className="branching-controls">
+      {answers.map((answer, index) => (
+        <li key={`branching-controls-answer-${index}`}>
+          <div className="fake-button">{answer.label}</div>
+          <NextStepArrow className="next-arrow" />
+          <select
+            className={(!answer?.next) ? 'next-is-submit' : ''}
+            data-index={index}
+            onChange={onChange}
+            value={answer?.next || ''}
+          >
+            <option
+              value={''}
+            >
+              Submit
+            </option>
+            {allSteps.map(([stepKey, stepBody]) => {
+              const taskKeys = stepBody?.taskKeys?.toString() || '(none)';
+              return (
+                <option
+                  key={`branching-controls-answer-${index}-option-${stepKey}`}
+                  value={stepKey}
+                >
+                  {taskKeys}
+                </option>
+              );
+            })}
+          </select>
+        </li>
+      ))}
+    </ul>
+  );
+}
+
+function NextStepArrow({
```

Comment:
This should probably be pulled out as an Icon / it's own file so it's usable in other parts of the codebase (if needed)